project/assets/common.less

```diff
@@ -0,0 +1,32 @@
+h3 {
+  margin-top: 0;
+}
+
+input {
+  border: none;
+  border-bottom: 1px solid lightgrey;
+  font-size: 18px;
+  margin-top: 30px;
+  margin-left: 40px;
+  text-align: left;
+  color: lightgrey;
+  line-height: 300%;
+}
+
+input-placeholder {
+  color: lightgrey;
+}
+
+nav {
+  display: inline;
+  background-color: white;
+  width: 1000px;
+  height: 100px;
+}
+
+a {
+  text-decoration: none;
+  font-size: 30px;
+  margin-left: 160px;
```

Comment:
this style is not generic enought to be part of the default a tag styling(as opposed to text-decoration which it is). Please apply a class where you need this kind of margin-left